betterproto/plugin.py

```diff
@@ -4,8 +4,11 @@
 import os.path
 import sys
 import textwrap
-from collections import defaultdict
-from typing import Dict, List, Optional, Type
+import re
+from typing import List
+
+import betterproto
```

Comment:
what's this for?

Reply:
see line 201
`re` is to match `*Value`-wrapper classes more precisely than before.
`betterproto` is imported to check if we have an actual implementation for that wrapping class.